src/main/kotlin/me/fornever/todosaurus/actions/OpenIssueInBrowserAction.kt

```diff
@@ -0,0 +1,35 @@
+package me.fornever.todosaurus.actions
+
+import com.intellij.openapi.actionSystem.ActionUpdateThread
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import kotlinx.coroutines.*
+import me.fornever.todosaurus.TodosaurusBundle
+import me.fornever.todosaurus.actions.extensions.getToDoTextRange
+import me.fornever.todosaurus.actions.extensions.tryActivateThisAction
+import me.fornever.todosaurus.services.ToDoItem
+import me.fornever.todosaurus.services.ToDoService
+
+class OpenIssueInBrowserAction : AnAction(
+    TodosaurusBundle.message("notification.openIssueInBrowser.action.name")
```

Comment:
_(Minor)_ Ideally, we shouldn't pass the text to the `AnAction` constructor and should rely on default localized text resolve.

Why? Because this allows IntelliJ to fetch the correct text without calling the action constructor — useful for features such as Find Action and in the action keymap listing.